src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -54,6 +88,33 @@ def value_change(self, value: float) -> str:
         value_str = "%.2f" % value
         return value_str
 
+    def setup_drive_switch_radio(self, title: str) -> QGroupBox:
+        """Create a radio button widget to switch between per-motor and velocity control modes
+
+        :param title: group box name
+        """
+        group_box = QGroupBox(title)
+        dbox = QVBoxLayout()
```

Comment:
I think it makes more sense to name this variable `vbox`

Reply:
What does vbox and dbox stand for anyways?

Reply:
`vbox` stands for vertical box, coming from the Qt docs: https://doc.qt.io/qt-6/qvboxlayout.html. Basically it's a layout object that lets you ensure that all objects that are part of this box are rendered on top of each other.

It looks like we call it `dbox` in some files but I'm not sure why. Might be a mistake.